lib/ProductOpener/Food.pm

```diff
@@ -3785,6 +3785,18 @@ sub mmoll_to_unit {
 		fi => "Ravintoarvosana",
 		ro => "Notă nutrițională",
 	},
+	"ibu" => {
+		de => "Internationale Bittereinheit",
+		en => "International Bittering Unit",
+		pt => "Medida do amargor internacional",
+		ru => "Международная единица горечи",
+		unit => "",
+	},
+	"alcohol-units" => {
+		de => "Alkoholeinheit",
+		fr => "Unité d'alcool",
```

Comment:
Should probably be plural. (same for German if it's not plural)

Reply:
fixed